blocks/features/features.css

```diff
@@ -0,0 +1,120 @@
+.features-wrapper {
+  padding: 0 var(--body-padding);
+}
+
+.features > div {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.features img {
+  width: 100%;
+}
+
+.features > div > div {
+  order: 1;
+}
+
+.features > div > .features-img-col {
+  order: 0;
+}
+
+.features > div > .features-img-col img {
+  display: block;
+}
+
+.features p {
+  color: var(--tab-description-color-light-grey);
+  line-height: 1.5;
+}
+
+.features p.features-tabs-hidden {
+  display: none;
+}
+
+.features ul {
+  padding: 0;
+}
+
+.features li {
+  padding: 0;
+}
+
+.features .features-tabs li > a {
+  color: var(--tab-header-color-blue);
+  font-size: var(--body-font-size-xs);
+  font-weight: var(--font-weight-bold);
+  position: relative;
+  display: inline-block;
+  letter-spacing: .004em;
+  cursor: pointer;
+  text-decoration: none;
+}
+
+.features .features-tabs li > a::after {
+  content: "";
+  position: relative;
+  display: inline-block;
+  width: 14px;
+  height: 8px;
+  margin: 0 3px;
+  background: url("../../../icons/tabs_arrow.svg") no-repeat;
+  background-size: 100%;;
+  transform: rotate(180deg);
+  transition: all .3s linear;
+}
+
+.features .features-tabs li > a.is-open::after {
+  transform: rotate(0)
+}
+
+.features .features-tabs li > a + em {
```

Comment:
The "improved" bagde is used the hero also the same way. li > a > em. Can you make it generally available?
![Screenshot 2023-08-04 at 10 42 07](https://github.com/hlxsites/bitdefender/assets/166901/2fbdc856-8dfd-4d94-8d27-dcdb021dd0d6)

Reply:
Perhaps you could implement this logic into lib-franklin.js which listens for a Tag inside of [] and assigns this it the correct style, for a general solution to this problem

```
export async function decorateTags(element) {
  const specialTags = ['[NEW]', '[EVOLVED]'];

  function replaceTagsInNode(node) {
    if (node.nodeType === Node.TEXT_NODE) {
      // This is a text node, check and replace tags
      let { nodeValue } = node;
      let replaced = false;

      // Capture all tags that follow the format [TAG]
      const tagRegex = /\[(.*?)\]/g;
      let match = tagRegex.exec(nodeValue);

      while (match !== null) {
        const fullTag = match[0];
        const tagContent = match[1];

        // Check if the tag is special (NEW or EVOLVED), otherwise it's considered IMPROVED
        const isSpecialTag = specialTags.includes(fullTag);
        const tagClass = isSpecialTag ? tagContent.toLowerCase() : 'improved';

        // Replace the tag with a span with a unique class
        nodeValue = nodeValue.replace(new RegExp(`\\${fullTag}`, 'g'), `<span class="tag tag-${tagClass}">${tagContent}</span>`);
        replaced = true;

        match = tagRegex.exec(nodeValue);
      }

      if (replaced) {
        const newNode = document.createElement('span');
        newNode.innerHTML = nodeValue;
        node.parentNode.replaceChild(newNode, node);
      }
    } else if (node.nodeType === Node.ELEMENT_NODE) {
      node.childNodes.forEach(replaceTagsInNode);
    }
  }

  replaceTagsInNode(element);
}
```

And this code into styles.css. Note with this implementation you will also have to change inside of header.css every selector that selects the spans inside of the mega-menu to only select the direct child span.

```
.tag {
  margin-left: 5px;
  color: var(--background-color);
  font-size: var(--body-font-size-xxxs);
  font-weight: var(--font-weight-boldest);
  line-height: 1;
  border-radius: 3px;
  padding: 2px 4px;
}

.tag-new {
  background-color: var(--new-tag);
}

.tag-evolved {
  background-color: var(--evolved-tag);
}

.tag-improved {
  background-color: var(--improved-tag);
}
```

Reply:
@johan-t There was a discussion around this, and eventually it was decided to tackle it as part of https://github.com/hlxsites/bitdefender/issues/96, to agree on a common authoring way for this tags/pills and then update the existing components once decided and implemented.

Reply:
@rhudea ok I did not know this in this case let me reopen my PR: https://github.com/hlxsites/bitdefender/pull/102

Reply:
Aslo, regarding hardcoding the tags in the JS...Not sure if this is the best authoring experience. What if they want to author a new tag which is not in the code? What if they misspell the tag and we give basically no indication of that? 

I would personally go for convention.